### PR TITLE
[Match] added installed certificate info logging

### PR DESCRIPTION
**Pull request by punksta**
Added feature https://github.com/fastlane/fastlane/issues/5014

```
[13:21:49]: Installing certificate...
+-------------------+-----------------------------------------------+
|                    Installed Code Certificate                     |
+-------------------+-----------------------------------------------+
| User ID           | **********                                    |
| Certificate Name  | iPhone Developer: Stas Shakirov (***********) |
| Organisation Unit | **********                                    |
| Organisation Name | ********* *****                               |
| Country           | US                                            |
| Start Datetime    | Jun  3 11:36:38 2016 GMT                      |
| End Datetime      | Jun  3 11:36:38 2017 GMT                      |
+-------------------+-----------------------------------------------+
[13:21:51]: Installing provisioning profile...
```

### CODE REVIEW

**Comment by KrauseFx** [comment]
This looks like a great PR, thanks for submitting this. Could you fix the code styling issues using `rubocop -a`? 👍

**Comment by punksta** [comment]
@KrauseFx fixed

File: match/lib/match/utils.rb [hunk 1]
```diff
@@ -9,6 +9,53 @@ def self.import(item_path, keychain)
       Helper.backticks(command, print: $verbose)
     end
 
+    # logs public key's  name, user, organisation, country, availability dates
+    def self.log_certificate_public_key(cer_certificate_path)
+      command = "openssl x509 -inform der -in #{cer_certificate_path.shellescape} -subject -dates -noout"
+      command << " &" # start in separate process
```
Comment (KrauseFx) [inline match/lib/match/utils.rb]: Why do we do this? Do we even receive the output in time here?

File: match/lib/match/utils.rb [hunk 2]
```diff
@@ -9,6 +9,53 @@ def self.import(item_path, keychain)
...
+      params = {
+          rows: table_data,
+          title: "Installed Code Certificate".green
+      }
+
+      puts ""
+      puts Terminal::Table.new(params)
+      puts ""
+      return table_data
+    end
```
Comment (KrauseFx) [inline match/lib/match/utils.rb]: Let's use this
```ruby
    ...
    return table_data
  rescue => ex
    UI.error(ex)
  end
```
We don't want `match` to crash when printing out a value

File: match/lib/match/utils.rb (hunk 1 above)
Comment (punksta) [inline match/lib/match/utils.rb]: Sorry, I do not understand you. What I need to change? Provision profiles are logged on every match's call. I write logging as that.

**Comment by punksta** [comment]
Also added setting of certificate name to env

**Comment by hjanuschka** [comment]
redone as https://github.com/fastlane/fastlane/pull/6907 - sorry rebase/merge was to hard.

**Comment by milch** [comment]
I'm closing this as we are merging this in #6907. 

Regardless, @punksta thank you for your contribution! 🚀

**Pull request closed without merging**